src/test/java/guitests/guihandles/InfoPanelHandle.java

```diff
@@ -100,9 +107,9 @@ public String getDebt() {
         return debtLabel.getText();
     }
 
-    public String getDateBorrow() {
-        return dateBorrowLabel.getText();
-    }
+    public String getDateBorrow() { return dateBorrowLabel.getText(); }
+
+    public String getDateRepaid() { return dateRepaidLabel.getText(); }
```

Comment:
how did these pass Travis
anw it's not supposed to be single line. same thing in PersonCardHandle.java

Reply:
Noted. Made the necessary changes.

Reply:
Noted. Made the necessary changes.